### PR TITLE
[Log] Wire server_log handler

### PR DESCRIPTION
**Pull request by lyrixx**
This is the last part of the new `server:log` command.

Maybe a release of the monolog bundle could be better to avoid to require a dev package.
ping @Seldaek 

WDYT?

### CODE REVIEW

**Comment by Seldaek** [comment]
monologbundle 3.1.0 is out

File: app/config/config_dev.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ monolog:
             type:   console
             process_psr_3_messages: false
             channels: ['!event', '!doctrine', '!console']
+        # To follow logs in real time, execute the following command:
+        # `bin/console server:log -vv`
```
Comment (kbond) [inline app/config/config_dev.yml]: Can we remove the need for `-vv`?

Comment (lyrixx) [inline app/config/config_dev.yml]: Actually, this is not mandatory ... it's just to get more logs.

Comment (kbond) [inline app/config/config_dev.yml]: Ah, I see, you only see warnings if excluded.

**Comment by lyrixx** [comment]
I rebased the PR. Now it required the new symfony/monolog-bundle package.

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**